docs/configuration/launch-parameters.mdx

```diff
@@ -202,6 +202,6 @@ It is runtime configs
 
 |                    Parameter                    |                      Default value                     |                                           Description                                           | Required | Example / Format |
 |:-----------------------------------------------:|:-------------------------------------------------------|:------------------------------------------------------------------------------------------------|:---------|:-----------------|
-| drill.probes.perf.mode                          | false                                                  | when it true, we use less memory but can lose coverage	                                         | no       | true/false       |
+| drill.probes.perf.mode                          | false                                                  | when it true, we use less memory but can lose coverage,also could affect on network performance between the agent and the admin and for real-time sessions between the admin and the fronted | no       | true/false        |
```

Comment:
```suggestion
| drill.probes.perf.mode                          | false                                                  | when it is true, we use less memory but can lose coverage, when it is false it could affect network performance between the agent and the admin and for real-time sessions between the admin and the fronted | no       | true/false        |
```
also or when it is false?

Reply:
oh yes, ofc when it false